website/docs/r/sql_source_representation_instance.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "google"
+page_title: "Google: google_sql_source_representation_instance"
+sidebar_current: "docs-google-sql-source-representation-instance"
+description: |-
+  Creates a new SQL Source Representation Instance, which holds metadata about
+  on-premises MySQL instances. This enables Google CloudSQL MySQL instances to
+  become replicas of on-premises databases.
+
+  Replicas are configured by setting the source representation instance name in
+  their `master_instance_name` value.
+
```

Comment:
Can you add a note here about the difference between a normal instance and a regular instance? Just for clarity, since I was a bit confused. Copy-paste from https://cloud.google.com/sql/docs/mysql/replication/external-server#config-description would be fine